src/machine/board_itsybitsy-nrf52840.go

```diff
@@ -0,0 +1,100 @@
+// +build itsybitsy_nrf52840
+
+package machine
+
+const HasLowFrequencyCrystal = true
+
+// GPIO Pins
+const (
+	D0  = P0_25 // UART TX
+	D1  = P0_24 // UART RX
+	D2  = P1_02
+	D3  = P0_06 // LED1
+	D4  = P0_20 // Button
```

Comment:
Perhaps P0_29 is correct.
https://cdn-learn.adafruit.com/assets/assets/000/087/158/original/adafruit_products_schem.png?1579387035

Reply:
Indeed: https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/variants/itsybitsy_nrf52840_express/variant.cpp#L33

Reply:
Changed.